PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/webapp/WebAppConfigFormPanelAdvance.java

```diff
@@ -0,0 +1,105 @@
+ *
+ * THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO
+ * THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+ * TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package com.microsoft.azure.toolkit.intellij.webapp;
+
+import com.intellij.openapi.project.Project;
+import com.microsoft.azure.management.appservice.OperatingSystem;
+import com.microsoft.azure.management.appservice.PricingTier;
+import com.microsoft.azure.toolkit.intellij.appservice.AppServiceInfoAdvancedPanel;
+import com.microsoft.azure.toolkit.intellij.appservice.AppServiceMonitorPanel;
+import com.microsoft.azure.toolkit.intellij.common.AzureFormPanel;
+import com.microsoft.azure.toolkit.lib.appservice.MonitorConfig;
+import com.microsoft.azure.toolkit.lib.appservice.Platform;
+import com.microsoft.azure.toolkit.lib.common.form.AzureFormInput;
+import com.microsoft.azure.toolkit.lib.webapp.WebAppConfig;
+import com.microsoft.azuretools.core.mvp.model.AzureMvpModel;
+import org.apache.commons.collections.ListUtils;
+
+import javax.swing.*;
+import java.util.List;
+
+public class WebAppConfigFormPanelAdvance extends JPanel implements AzureFormPanel<WebAppConfig> {
+    private JTabbedPane tabPane;
+    private JPanel pnlRoot;
+    private Project project;
+    private AppServiceInfoAdvancedPanel<WebAppConfig> appServiceConfigPanelAdvanced;
+    private AppServiceMonitorPanel appServiceMonitorPanel;
+    private JPanel pnlMonitoring;
+    private JPanel pnlAppService;
+
+    public WebAppConfigFormPanelAdvance(final Project project) {
+        this.project = project;
+    }
+
+    @Override
+    public void setVisible(final boolean visible) {
+        pnlRoot.setVisible(visible);
+    }
+
+    @Override
+    public WebAppConfig getData() {
+        final WebAppConfig data = appServiceConfigPanelAdvanced.getData();
+        data.setMonitorConfig(appServiceMonitorPanel.getData());
+        return data;
+    }
+
+    @Override
+    public void setData(final WebAppConfig data) {
+        appServiceConfigPanelAdvanced.setData(data);
+        appServiceMonitorPanel.setData(data.getMonitorConfig());
+    }
+
+    @Override
+    public List<AzureFormInput<?>> getInputs() {
+        return ListUtils.union(appServiceConfigPanelAdvanced.getInputs(), appServiceMonitorPanel.getInputs());
+    }
+
+    public void setDeploymentVisible(final boolean visible) {
+        this.appServiceConfigPanelAdvanced.setDeploymentVisible(visible);
+    }
+
+    public void setValidPricingTier(final List<PricingTier> validPricingTiers, final PricingTier defaultPricingTier) {
+        this.appServiceConfigPanelAdvanced.setValidPricingTier(validPricingTiers, defaultPricingTier);
```

Comment:
how about making `setItems` public? and use `setValue` to set the default value?

Reply:
`setItems` values will lose after `refresh`, though we didn't trigger `refresh` after initialization now, but I think it may be better to change the behavior of `loadItems` in case changes in future